Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Sandboxed/SelectorsPollerScript.js

```diff
@@ -47,7 +47,8 @@ window.__firefox__.execute(function($) {
   let notYetQueriedIds = []
   
   const CC = {}
-  CC.allSelectorsToRules = CC.allSelectorsToRules || new Map()
+  CC.allHideSelectorsToRules = CC.allHideSelectorsToRules || new Map()
+  CC.allRules = []
   CC.observingHasStarted = CC.observingHasStarted || false
```

Comment:
Same thing here. `CC` is initialized on line 49, so it'll never have an `observingHasStarted`. Just do: `CC.observingHasStarted = false`.

Reply:
Same answer. The original file was identical to the one in desktop but it has since diverged.